client/sites/settings/controller.tsx

```diff
@@ -1,47 +1,48 @@
 import { __ } from '@wordpress/i18n';
-import makeSidebar, { PanelWithSidebar } from '../components/panel-sidebar';
+import { useSelector } from 'react-redux';
+import AdministrationSettings from 'calypso/my-sites/site-settings/administration';
+import AgencySettings from 'calypso/my-sites/site-settings/agency';
+import CachesSettings from 'calypso/my-sites/site-settings/caches';
+import SiteSettings from 'calypso/my-sites/site-settings/site';
+import WebServerSettings from 'calypso/my-sites/site-settings/web-server';
+import isSiteWpcomStaging from 'calypso/state/selectors/is-site-wpcom-staging';
+import { useSelectedSiteSelector } from 'calypso/state/sites/hooks';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { SidebarItem, Sidebar, PanelWithSidebar } from '../components/panel-sidebar';
 import type { Context as PageJSContext } from '@automattic/calypso-router';
 
-const SettingsSidebar = makeSidebar( {
-	items: [
-		{
-			key: 'site',
-			get label() {
-				return __( 'Site' );
-			},
-		},
-		{
-			key: 'administration',
-			get label() {
-				return __( 'Administration' );
-			},
-		},
-		{
-			key: 'agency',
-			get label() {
-				return __( 'Agency' );
-			},
-		},
-		{
-			key: 'caches',
-			get label() {
-				return __( 'Caches' );
-			},
-		},
-		{
-			key: 'web-server',
-			get label() {
-				return __( 'Web Server' );
-			},
-		},
-	],
-} );
+export function SettingsSideBar( { selectedItemKey }: { selectedItemKey: string } ) {
```

Comment:
Let's keep the name as `SettingsSidebar` (lowercase `b`) for consistency 🙏

Reply:
Typo